chaincode/compute_plan.go

```diff
@@ -311,13 +311,45 @@ func getComputePlan(db *LedgerDB, key string) (resp outputComputePlan, err error
 		}
 	}
 
+
+	status, _ := getComputePlanStatus(db, tupleKeys)
```

Comment:
We should check the error here even for the status computation. Also I think it would be good to return the compute plan at creation.